content/post/21-10-01-progress-report-27.md

```diff
@@ -0,0 +1,281 @@
+
+{{< video-auto "/video/pr_27/chess.mp4" >}}
+
+Zumorica further improved upon Visne's code making it easy to add different tabletop games, adding the game of parchís in the process!
+
+{{< imgw "/images/post/pr_27/sorry" >}}
+
+### Alt-Clicking
+*Contributed by ElectroSR*
+
+Alt+E, Alt+Z and Alt+Left-Mouse keybindings can trigger alternative interactions on entities. Some existing context-menu actions are now considered "alternative interactions", for example ejecting IDs/beakers/magazines or locking & unlocking lockers can be done via these new keybindings.
+
+{{< imgw "/images/post/pr_27/alt" >}}
+
+### Wielding
+*Contributed by mirrorcult*
+
+You can now wield (two-hand) some weapons like the fire-axe which will increase the damage per swing. The fire-axe has also received a buff against windows and wooden objects.
+
+{{< video-auto "/video/pr_27/wielding.mp4" >}}
+
+### Ambient sounds
+*Contributed by metalgearsloth, Swept*
+
+Ambient sounds can now play from objects around the station for improved immersion. APCs hum, vending machines whirr, and so on.
+
+<!-- TODO: Align Left -->
+{{< video-audio "/video/pr_27/ambience.mp4" >}}
+
+### Vending Machine Ads
+*Contributed by Moses, Visne and Zumorica*
+
+{{< imgw "/images/post/pr_27/ads" >}}
+
+### Clothing Storage Visualizers!
+
+*Contributed by Ygg01 and Swept*
+
+Certain items when placed in belts show a sprite on the belt icon itself.
+
+<!-- TODO: Align Left -->
+{{< video-auto "/video/pr_27/belts.mp4" >}}
+
+A lot of belts have gotten sprites for this feature, as well as the botany belt being added to give botanists a helping hand.
+
+{{< imgw "/images/post/pr_27/belts" >}}
+
+### Meth
+*Contributed by scaly-chimp*
+
+Scaly-chimp broke bad and added meth so aspiring chemists can recreate scenes from the hit TV show Big Bang Theory.
+
+<!-- Insert video of someone high on meth -->
+
+### UI for setting solution transfer amount
+*Contributed by mirrorcult*
+
+Entities likes beakers and bottles now have a verb that opens a UI which allows you to set the transfer amount.
+
+{{< imgw "/images/post/pr_27/transfer" >}}
+
+### Examineable Batteries
+*Contributed by 20kdc*
+
+When examining batteries you can see their specific charge value.
+
+<!-- TODO: Align Left -->
+{{< video-auto "/video/pr_27/cellexamine.mp4" >}}
+
+### Multiple Viewports
+*Contributed by PJB*
+
+Moving this to here so we can prepare stuff like camera demos or chat window demos to really sell this feature.
+
+### Windoors
+*Contributed by mirrorcult*
+
+NanoTrasen has finally found space in the budget to install windoors around the station, increasing security and safety!
+
+{{< imgw "/images/post/pr_27/windoors" >}}
+
+### Shuttles
+*Contributed by metalgearsloth, Acruid, and 20kdc*
+
+The first iteration of shuttle driving has made it into the game. Currently it allows you to drive them around in a strafing manner, with rotation planned once the technical blockers have been overcome in the coming months.
```

Comment:
afaict shuttle rotation is in now, just not over the pilot console

Reply:
Since that's not accessible by players on pilot console i'd say it's not worth saying